build_tools/pkgci/build_tests_using_package.sh

```diff
@@ -53,11 +53,11 @@ fi
 if (( IREE_METAL_DISABLE == 1 )); then
   cmake_config_options+=("-DIREE_HAL_DRIVER_METAL=OFF")
 fi
-if (( IREE_CUDA_DISABLE == 1 )); then
-  cmake_config_options+=("-DIREE_HAL_DRIVER_CUDA=OFF")
+if (( IREE_CUDA_DISABLE == 0 )); then
+  cmake_config_options+=("-DIREE_HAL_DRIVER_CUDA=ON")
```

Comment:
Checking the negative here is error prone / confusing if the environment variable is unset:

* `IREE_CUDA_DISABLE` unset --> CUDA disabled
* `IREE_CUDA_DISABLE` set to 0 --> CUDA enabled
* `IREE_CUDA_DISABLE` set to 1 --> CUDA disabled

I think we should flip this to `IREE_CUDA_ENABLE`:

* `IREE_CUDA_ENABLE` unset --> CUDA disabled
* `IREE_CUDA_ENABLE` set to 0 --> CUDA disabled
* `IREE_CUDA_ENABLE` set to 1 --> CUDA enabled

Same for `IREE_HIP_DISABLE`.